admin/src/admin.js

```diff
@@ -214,7 +214,10 @@ const HiddenAppBar = withStyles({
       display: "none"
     }
   }
-})(props => <AppBar {...props} className={props.classes.hideOnDesktop} />);
+})(props => {
+  const { classes, ...other } = props;
+  return <AppBar {...other} className={classes.hideOnDesktop} />;
```

Comment:
Why do we no longer want to pass classes down?

Reply:
Throwing the error in console from Material UI, styles still work as intended with the new syntax.